tests/test_skvbc_linearizability.py

```diff
@@ -45,11 +45,29 @@ def start_replica_cmd(builddir, replica_id):
             ]
 
 
+def tracker_decorator():
```

Comment:
Let's move this into the skvbc_history_tracker.py module, so that other tests may also use it.